lib/daru/view/adapters/highcharts/display.rb

```diff
@@ -1,10 +1,10 @@
 require_relative 'layout_helper_iruby'
 require_relative 'iruby_notebook'
+require 'daru/view/constants'
 
 module LazyHighCharts
   def self.init_script(
-    dependent_js=['highstock.js', 'highcharts-more.js', 'modules/exporting.js',
-                  'highcharts-3d.js', 'modules/data.js']
+    dependent_js=[HIGHSTOCK, MAP, EXPORTING, HIGHCHARTS_3D, DATA]
```

Comment:
`[HIGHSTOCK, MAP, EXPORTING, HIGHCHARTS_3D, DATA]` is also constant as web app highcharts dependencies.